app/src/main/res/layout/drag_drop_test_fragment.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+  <androidx.recyclerview.widget.RecyclerView
+    android:id="@+id/drag_drop_recycler_view"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"/>
+</layout>
```

Comment:
Please add EOF newline.

Reply:
updated.

Reply:
It seems to still be missing per the latest code.

Reply:
Updated.